cookbooks/delayed_job4/README.md

```diff
@@ -0,0 +1,42 @@
+# Delayed Job
+
+This cookbook can serve as a good starting point for setting up Delayed Job support in your application.
+In this recipe your Delayed Job workers will be set up to run under monit. The number of workers will
+vary based on the size of the instance running Delayed Job.
+
+** Please Note ** This recipe will setup delayed_job on a single instance environment or on a Utility instance in a cluster environment. If you need delayed_job to run on app instances (if you are not using a Utility instance), you will need to modify the recipe.
+
+## Installation
```

Comment:
Installation instructions need to be updated for stable-v5
e.g. https://github.com/engineyard/ey-cookbooks-stable-v5/tree/next-release/cookbooks/redis#installation